File: 349-intersection-of-two-arrays/intersection-of-two-arrays.py
class Solution:
    def intersection(self, nums1: List[int], nums2: List[int]) -> List[int]:
        dic = []
        for i in range(len(nums1)):
            for j in range(len(nums2)):
                if nums1[i] == nums2[j]:
                    dic.append(nums1[i])
        return list(set(dic))
        

File: 349-intersection-of-two-arrays/intersection-of-two-arrays.php
class Solution {

    /**
     * @param Integer[] $nums1
     * @param Integer[] $nums2
     * @return Integer[]
     */
    function intersection($nums1, $nums2) {
        $map = array();
        
        /* for($i=0; $i<sizeof($nums1); $i++){
            for($j=0; $j<sizeof($nums2); $j++){
                if(($nums[$i] == $nums[$j] )){
                    array_push($map , $nums[$j]);
                }
            }
        } */
        return array_unique(array_intersect($nums1 , $nums2));
        
    }
}